src/Agent/NewRelic/Agent/Extensions/Providers/Wrapper/AwsLambda/HandlerMethodWrapper.cs

```diff
@@ -120,6 +120,9 @@ public object GetInputObject(InstrumentedMethodCall instrumentedMethodCall)
         public bool IsTransactionRequired => false;
 
         private static bool _coldStart = true;
+        private HashSet<string> _unexpectedResponseTypes = new();
+        private HashSet<string> _unsupportedInputTypes = new();
```

Comment:
Do we need to use thread safe collections here?

Reply:
I don't think so; worst case, we log the warning a couple of times instead of just once.

Reply:
I'm ok if we warn a few times, but I'm more concerned about the internal state of the collection being in a bad state if add is called concurrently.